client/js/options.js

```diff
@@ -1,6 +1,7 @@
 "use strict";
 
 const $ = require("jquery");
+const _ = require("lodash");
```

Comment:
Ah I suspected something like that should exist, was looking for it. Thanks will do that.